dependencies.yaml

```
@@ -424,3 +424,22 @@ dependencies:
     refPaths:
     - path: images/build/setcap/variants.yaml
       match: '[a-zA-Z]+\-v((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)'
+
+# Build environments
+  - name: "gcr.io/k8s-testimages/gcb-docker-gcloud"
+    version: v20210722-085d930
+    refPaths:
+    - path: cmd/vulndash/cloudbuild.yaml
+      match: "name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v([0-9]{8})-([a-z0-9]{7})'"
```

nit, the groups are not required:
```suggestion
    - path: cmd/vulndash/cloudbuild.yaml
      match: "name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v[0-9]{8}-[a-z0-9]{7}'"
```

---

Great point! Fixed!